zcnbridge/config.go

```diff
@@ -24,43 +29,66 @@ type BridgeSDKConfig struct {
 }
 
 type BridgeClient struct {
+	KeyStore
+	transaction.TransactionProvider
+	EthereumClient
```

Comment:
We'd better keep the members private

Reply:
We cannot, becase in wasmsdk package we explicetely initialize BridgeClient without any factory method